.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,32 @@
+name: PR CI
+
+on:
+  pull_request:
+    branches:
+      - 'mayastor'
+    paths-ignore:
```

Comment:
Doesn't this cause issues setting `verify-lint-and-build` as required check to pass o branch protection?

Reply:
Issues as in? Seems to work on the PR atleast? Are you suggesting to use the dorny thing?

Reply:
We had some issues with protected branches preventing PRs from merging because the ignore means the check doesn't run, so the protection prevents PR from merging.